### PR TITLE
feature: Add Replay Buffers

### PR DESCRIPTION
**Pull request by sshcrack**
This pull request adds the replay buffer feature requested in #47. The PR [libobs_sharp/#1](https://github.com/lulzsun/libobs-sharp/pull/1) has to be merged first so the method `obs_output_get_id ` can be used.

### CODE REVIEW

File: Classes/Recorders/PlaysLTCRecorder.cs [hunk 1]
```diff
@@ -123,5 +123,9 @@ public class PlaysLTCRecorder : BaseRecorder {
 
         public override void GainedFocus() {
         }
+
+        public override bool? TrySaveReplayBufferAndBookmarks() {
+
```
Comment (sshcrack) [inline Classes/Recorders/PlaysLTCRecorder.cs:128]: Not sure here. Does PlaysLTC have such an feature? Are we still supporting it?

File: Classes/Utils/Helpers.cs [hunk 2]
```diff
@@ -370,8 +370,9 @@ public static class Functions {
                 }
             }
 
+            var details = "";
```
Comment (sshcrack) [inline Classes/Utils/Helpers.cs]: Was just for debugging purposes, reverted the changes in a commit later

**Comment by sshcrack** [comment]
oh, also I accidentally left a whitespace in the `libobs-sharp.example` project. [This](https://github.com/lulzsun/libobs-sharp/pull/2) pull request fixes the issue.

File: Classes/Recorders/PlaysLTCRecorder.cs (hunk 1 above)
Comment (Segergren) [reply Classes/Recorders/PlaysLTCRecorder.cs:128]: What do you think @lulzsun? Time to remove it :)

Comment (lulzsun) [reply Classes/Recorders/PlaysLTCRecorder.cs:128]: We are technically no longer supporting PlaysLTC, so we could see it being removed in the future. But let's just pretend to support this for now and deal with it later.

**Comment by sshcrack** [comment]
Hey, it's been a while. Are we ready to merge?

**Comment by Segergren** [comment]
Should we create a new stable version and then merge this @lulzsun?

**Review by lulzsun — APPROVED** [review]

**Comment by lulzsun** [comment]
Hi, sorry I was busy. I planned to test this last week but ended up forgetting, so I just took some time right now to try it out.

It looks like it works well, so I'll be merging this right away. I think I want to create a stable version with this feature in as it is one of the mostly requested features, but let's have other users test it out in nightly before we do so.

Thanks again for the pull request!

**Comment by sshcrack** [comment]
Were there any issues? If not are we ready for a stable release?

**Comment by Segergren** [comment]
@sshcrack I suggest disabling fragmented output options as they are not compatible, see #207 👍

**Pull request merged**